src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -1287,7 +1262,7 @@ void SizeEvapCooler(EnergyPlusData &state, int const EvapCoolNum)
             }
         }
     } else if (CurZoneEqNum > 0) { // zone equipment
-        if (!IsAutoSize && !SizingDesRunThisAirSys) {
+        if (!IsAutoSize && !SizingDesRunThisZone) {
```

Comment:
And here is another change from original. If these are the cause of diffs I'll change back and open an issue.